gengokit/handler/handler.go

```diff
@@ -0,0 +1,315 @@
+// package handler parses service handlers and add/removes exported methods to
```

Comment:
May be more straightforward to write as:

    Package handler manages the exported methods in the service handler code, adding/removing
    exported methods to match the defined service methods.